whiteblock/cmd/get.go

```diff
@@ -304,22 +307,73 @@ func getBlockCobra(cmd *cobra.Command, args []string) {
 	}*/
 }
 
+func getBlockHeightByNode(cmd *cobra.Command, args []string) {
+	wg := sync.WaitGroup{}
+	mux := sync.Mutex{}
+
+	util.CheckArguments(cmd, args, 0, 1)
+
+	if args[0] == "all" {
```

Comment:
It should be `wb get block number --all` not wb get block number all, also this will panic. Switch to using a flag.

Reply:
I am using a flag